src/Search/Request/RequestConfiguration.php

```diff
@@ -82,6 +84,41 @@ public function getPage(): int
         return (int) $this->request->get('page', 1);
     }
 
+    /**
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+     */
+    public function manageRangeField(string $field): void
+    {
+        $price = $this->request->get($field, []);
+        if (!\is_array($price) || empty($price)) {
+            return;
+        }
+
+        /** @var array $price */
+
+        // Reverse min and max if min is greater than max
+        if (isset($price['min'], $price['max'])) {
+            $min = (int) $price['min'];
+            $max = (int) $price['max'];
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** rename the `price` variable to `range`‏

Reply:
Well seen !